docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -89,88 +85,85 @@ on GPUs.
-| AMD PRO W7900            | `w7900`     | `gfx1100`           | `rdna3`                |
-| AMD PRO W7800            | `w7800`     | `gfx1100`           | `rdna3`                |
-| AMD RX7800XT             | `rx7800xt`  | `gfx1101`           | `rdna3`                |
-| AMD RX7700XT             | `rx7700xt`  | `gfx1101`           | `rdna3`                |
-| AMD PRO V710             | `v710`      | `gfx1101`           | `rdna3`                |
-| AMD PRO W7700            | `w7700`     | `gfx1101`           | `rdna3`                |
-
-For a more comprehensive list of prior GPU generations, you can refer to the
-[LLVM AMDGPU backend](https://llvm.org/docs/AMDGPUUsage.html#processors).
-
-The `iree-hip-target` option support three schemes:
-
-1. The exact GPU product (SKU), e.g., `--iree-hip-target=mi300x`. This allows
-  the compiler to know about both the target architecture and about additional
-  hardware details like the number of compute units. This extra information
-  guides some compiler heuristics and allows for SKU-specific [tuning
-  specs](../../reference/tuning.md).
-2. The GPU architecture, as defined by LLVM, e.g., `--iree-hip-target=gfx942`.
-  This scheme allows for architecture-specific [tuning
-  specs](../../reference/tuning.md) only.
-3. The architecture code name, e.g., `--iree-hip-target=cdna3`. This scheme gets
-  translated to closes matching GPU architecture under the hood.
-
-We support for common code/SKU names without aiming to be exhaustive. If the
-ones you want are missing, please use the GPU architecture scheme (2.) as it is
-the most general.
+???+ tip "Tip - HIP bitcode files"
+
+    That IREE comes with bundled bitcode files, which are used for linking
+    certain intrinsics on AMD GPUs. These will be used automatically or if the
+    `--iree-hip-bc-dir` is empty. As additional support may be needed for
+    different chips, users can use this flag to point to an explicit directory.
+    For example, in ROCm installations on Linux, this is often found under
+    `/opt/rocm/amdgcn/bitcode`.
+
+???+ tip "Tip - HIP targets"
+
+    A HIP target (`iree-hip-target`) matching the LLVM AMDGPU backend is needed
+    to compile towards each GPU chip. Here is a table of commonly used
+    architectures:
+
+    | AMD GPU                  | SKU Name    | Target Architecture | Architecture Code Name |
+    | ------------------------ | ----------- | ------------------- | ---------------------- |
+    | AMD MI100                | `mi100`     | `gfx908`            | `cdna1`                |
+    | AMD MI210                | `mi210`     | `gfx90a`            | `cdna2`                |
+    | AMD MI250                | `mi250`     | `gfx90a`            | `cdna2`                |
+    | AMD MI300X (early units) | N/A         | `gfx940`            | `cdna3`                |
+    | AMD MI300A (early units) | N/A         | `gfx941`            | `cdna3`                |
```

Comment:
We probably want to drop these: https://github.com/llvm/llvm-project/pull/125836

Reply:
Sure. Maybe in a separate change? Together with updates to https://github.com/iree-org/iree/blob/f3bef2de123f08b4fc3b0ce691494891bd6760d0/compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp#L322-L323

Reply:
We can keep it around in the code but not advertise on the website until the llvm change lands